### DISCUSSION TITLE
I/flutter ( 4815): error is:[firebase_auth/channel-error] Unable to establish connection on channel.

### DISCUSSION THREAD

**Discussion by dinshad123** [firebase/flutterfire#11930] [Q&A]
Hai team i am using firbase_auth in my project. When i use   createUserWithEmailAndPassword method to register my user i am getting an error that "I/flutter ( 4815): error is:[firebase_auth/channel-error] Unable to establish connection on channel.".I have tried to so many thing like flutter clean and flutter pub get,flutter pub upgrade,also tried to downgrade firebase auth and core.Still i am facing this issue.Can't getting my user credentials in firbase console.

**Comment by Shahabmustafa** [comment]
void main() async {
  WidgetsFlutterBinding.ensureInitialized();
  await Firebase.initializeApp();
  runApp(const MyApp());
}
